.github/workflows/recreate-linux-runners.yml

```
@@ -1,17 +1,21 @@
 name: Recreate Linux self-hosted runners on schedule
 
 on:
-  workflow_dispatch:
+  push:
+    paths:
+      - .github/workflows/recreate-linux-runners.yml
   schedule:
     # Once each 24 hours, at 1 during the night
     - cron: "0 1 * * *"
+  workflow_dispatch:
```

does this work? How about 
```suggestion
  workflow_dispatch
```

---

Without the colon it throws a syntax error. I used the GitHub workflow YAML schema when putting this together. autopublish.yml also uses `workflow_dispatch:`